components/integrations/CalendarsList.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { ReactNode } from "react";
+
+import { List } from "@components/List";
+import Button from "@components/ui/Button";
+
+import ConnectIntegration from "./ConnectIntegrations";
+import IntegrationListItem from "./IntegrationListItem";
+
+interface Props {
+  calendars: {
+    children?: ReactNode;
+    description: string;
+    imageSrc: string;
+    title: string;
+    type: string;
+  }[];
```

Comment:
Can add a notifier here to extract the bespoke handling further down

```suggestion
  }[];
  onChanged: () => Promise<void> | void
```